index.css

```diff
@@ -82,3 +81,149 @@ body {
 .social-media-spacer {
   margin-left: 20px;
 }
+
+.grid-item {
+  display: block;
+  box-shadow: 2px;
+  padding: 5px;
+  width: 300px;
+  border: 2px solid#DFE1E6;
+  border-radius: 10px;
+}
```

Comment:
Please the content of the project should be aligned with Flex to ensure a good responsive layout in your project 😄